run.sh

```diff
@@ -17,5 +17,5 @@ __dotnet=$__toolsLocalPath/dotnetcli/dotnet
 
 cp -fR $__scriptpath/tools-override/* $__toolsLocalPath 
 
-$__dotnet $__toolsLocalPath/run.exe $*
-exit $?
\ No newline at end of file
+DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1 $__dotnet $__toolsLocalPath/run.exe $*
```

Comment:
I added it to the Windows scripts, and to init-tools as well. There are multiple ways to build things (e.g. through the scripts or using MSBuild directly), so I wanted to make sure the first-time experience never ran, regardless of what command was used..